Source/WebCore/Modules/webxr/WebXROpaqueFramebuffer.h

```diff
@@ -35,6 +35,10 @@
 #include <wtf/Ref.h>
 #include <wtf/RetainPtr.h>
 
+#if USE(IOSURFACE_FOR_XR_LAYER_DATA) || USE(MTLTEXTURE_FOR_XR_LAYER_DATA)
+#define USE_WEBXR_USE_EGL_IMAGE 1
```

Comment:
It's a bit inconvenient to signal that "this is a standard USE() macro" but then at the same time not have it in standard place, available for all .cpp files, and at the same time not having it confied to just one .cpp file. 
So maybe this could be just omitted and written inline like
`#if USE(IOSURFACE_FOR_XR_LAYER_DATA) || USE(MTLTEXTURE_FOR_XR_LAYER_DATA)`.

There's 4 uses of USE_WEBXR_USE_EGL_IMAGE So if you remove the definition, you have penalty of score 3.

If I understand correctly, the target is to replace
`USE(IOSURFACE_FOR_XR_LAYER_DATA) || USE(MTLTEXTURE_FOR_XR_LAYER_DATA`
with
`PLATFORM(COCOA)`..